index.html

```diff
@@ -6,7 +6,7 @@
 -->
 <html>
 	<head>
-		<title>PiVPN: Simplest setup of OpenVPN</title>
```

Comment:
Maybe change to "PIVPN: Simplest way to setup a vpn"  .. What you think?

Reply:
sounds good.